### PR TITLE
9.24 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

File: content/en/docs/releasenotes/studio-pro/9/9.24.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+* We fixed the start-up of the console log while running from Eclipse.
+* We fixed an error that appeared after a validation message in the **Properties** dialog box of a workflow activity.
+
+### Deprecations
+
+In the [Runtime API](/apidocs-mxsdk/apidocs/runtime-api/), we did the following:
+
+* We deprecated `Core.isSubClassOf` method variants. You can instead use the `IMendixObject.isInstanceOf(IMetaObject)`, `IMendixObject.isInstanceOf(String)`, `IMetaObject.isSubClassOf(IMetaObject)`, or `IMetaObject.isSubClassOf(String)` methods to achieve the same result.
+* We deprecated the setter methods that were for internal use. We have provided constructors as an alternative for these methods whereever applicable.
+* We deprecated the `com.mendix.core.Core.deleteAsync` methods. You can use the synchronous version or run the synchronous version in a thread if needed. In addition, we introduced a delete method with events and with the `useDeleteBehavior` parameter as a replacement for calling `deleteAsync` with `useDeleteBehavior` set to false.
```
Comment (atennapel) [inline content/en/docs/releasenotes/studio-pro/9/9.24.md]: We (Runtime Core team) would like to adjust this change by changing "run the synchronous version in a thread if needed" to "run the synchronous version in a task queue if needed"

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.24.md]: Updated, thanks: https://github.com/mendix/docs/pull/5759/commits/4094f7878ac0b1d035863b1bd31dd87c4f0cd007

Comment (atennapel) [reply content/en/docs/releasenotes/studio-pro/9/9.24.md]: Thanks!

File: content/en/docs/releasenotes/studio-pro/9/9.24.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+We made a major update to your business events modelling experience:
+
+* Business event service creators and users can now configure multiple event publishers and subscribers. This means you get the full power of asynchronous communication across your enterprise landscape.
+* You can now create business events within the service document itself. Based on the definition of the events and the implementation, entities are then created for you automatically.
+* Published business event service and consumed business event service options have been combined into one menu item to help you get going quickly. [**TW**]
+
+#### Other Improvements
+
+* Studio Pro now now retrieves non-persistable objects over a direct association without sending a request as long as they have no virtual attributes.
```
Comment (Robinfr) [inline content/en/docs/releasenotes/studio-pro/9/9.24.md]: The client does this, not Studio Pro.

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.24.md]: Changed, thanks: https://github.com/mendix/docs/pull/5759/commits/6bca2bc4aef9b92954d3ed1e77279ce2514a10aa

**Pull request merged**